webapp/src/Controller/API/MetricsController.php

```diff
@@ -101,7 +101,7 @@ public function prometheusAction(Request $request): Response
             /** @var Submission[] $submissions */
             list($submissions, $submissionCounts) = $this->submissionService->getSubmissionList([$contest->getCid() => $contest], [], 0);
             foreach ($submissionCounts as $kind => $count) {
-                $m['submissions_' . $kind]->set($count, $labels);
+                $m['submissions_' . $kind]->set((float)$count, $labels);
```

Comment:
Why is this a float?

Reply:
Int is better. I think it is string currently